### ISSUE TITLE
Make `getConf` and `getAllConf` Ice calls return actually used configuration

### ISSUE THREAD

**Issue by Natenom** [mumble-voip/mumble#1174]
If defaultchannel is not set the user lands in the root channel (id 0), so the default value for this variable is 0.
So for this case getConf() should also return 0 instead of ''.

**Comment by ZeroAbility** [comment]
Was this ever implemented?

**Comment by Krzmbrzl** [comment]
No. And looking at the code, the current behavior is such that this call will only query configurations that have been set on the server explicitly (e.g. via Ice). Configuration values that were set in the INI file don't seem to be returned either as those are (afaik) not written into the DB.
Same goes for the `getAllConf` Ice call.

However, I think the argument is definitely in favor of changing this behavior and instead returning the configuration that the server is currently using and not only what is currently stored in the database (after all, what is stored in the DB and what was loaded from the INI file is essentially an implementation detail that Ice users shouldn't have to care about).
